en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/service-discovery/choreo-connect-and-k8s.md

```diff
@@ -0,0 +1,48 @@
+# Configure Choreo Connect with Consul service mesh deployed on Kubernetes
+Example:
```

Comment:
Please add some kind of explanation or overview that covers the following:

- What are we trying to achieve with this documentation.
- Description of what Consul service mesh is.
- Why someone would want to connect Choreo Connect with Consul.